processor/otel/consumer.go

```diff
@@ -640,16 +641,64 @@ func convertSpanEvent(
 	transaction *model.Transaction, span *model.Span, // only one is non-nil
 	out *model.Batch,
 ) {
+	var e *model.Error
+	isJaeger := strings.HasPrefix(metadata.Service.Agent.Name, "Jaeger")
+	if isJaeger {
+		e = convertJaegerErrorSpanEvent(logger, event)
+	} else {
+		// Translate exception span events to errors.
+		//
+		// If it's not Jaeger, we assume OpenTelemetry semantic conventions.
+		//
+		// TODO(axw) we don't currently support arbitrary events, we only look
+		// for exceptions and convert those to Elastic APM error events.
+		if event.Name() != "exception" {
+			// `The name of the event MUST be "exception"`
+			return
+		}
+		var exceptionEscaped bool
+		var exceptionMessage, exceptionStacktrace, exceptionType string
+		event.Attributes().ForEach(func(k string, v pdata.AttributeValue) {
+			switch k {
+			case conventions.AttributeExceptionMessage:
+				exceptionMessage = v.StringVal()
+			case conventions.AttributeExceptionStacktrace:
+				exceptionStacktrace = v.StringVal()
+			case conventions.AttributeExceptionType:
+				exceptionType = v.StringVal()
+			case "exception.escaped":
+				exceptionEscaped = v.BoolVal()
+			}
+		})
+		if exceptionMessage == "" && exceptionType == "" {
+			// `At least one of the following sets of attributes is required:
+			// - exception.type
+			// - exception.message`
+			return
```

Comment:
This gets translated into `error.exception`, for which the elastic apm intake protocol requires a `message`. Below, if no `message` is present you set a default value - I think it is fair, to include as many exceptions from otel as possible. 

I am not clear on why you require a `type` or a `message` though? The requirement for a `message` or `type` on the elastic apm intake protocol only applies to `error.log` - but this is not set here.

Reply:
The "At least one of..." comment comes from https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/exceptions.md. I'll be more explicit about that.